lib/expense_tracker_web/live/dashboard_live/form_component.ex

```diff
@@ -0,0 +1,89 @@
+defmodule ExpenseTrackerWeb.DashboardLive.FormComponent do
+  use ExpenseTrackerWeb, :live_component
+
+  alias ExpenseTracker.Expenses
+  alias ExpenseTracker.Expenses.Expense
```

Comment:
```suggestion
  alias YoloPenny.Expenses.Expense
```
It seems you did copy paste from the previous line here. I couldn't run the server because of the wrong module name.